xilem_web/src/events.rs

```diff
@@ -150,11 +153,14 @@ where
     })
 }
 
-#[allow(clippy::too_many_arguments)] // reason: This is only used to avoid more boilerplate in macros, also so that rust-analyzer can be of help here.
+#[allow(
+    clippy::too_many_arguments,
```

Comment:
Hmm, why is this still needed?

Reply:
Oh right, we allow more arguments now IIRC. I removed this.